ui/app/reports/views/reports.html

```diff
@@ -45,7 +45,7 @@ <h2 class="section-title">{{::'REPORTS_TITLE_KEY' | translate}}</h2>
             <tr ng-repeat="report in ::reportsRequiringDateRange" show-if-privilege="{{:: report.requiredPrivilege}}">
                 <td>{{:: report.name |translate }}</td>
                 <td class="reports-start-date">
-                    <input date-converter class="form-field start-date" type="date" max="{{report.stopDate | date:'yyyy-MM-dd'}}"
+                    <input date-converter class="form-field start-date" type="date" max="{{report.stopDate + 1 | date:'yyyy-MM-dd'}}"
```

Comment:
I didn't understand. What does this code change do. Also, it says "stopDate" + 1 for "startDate"? Is this as expected?

Reply:
Does the max provide some check to ensure startDate is always <= stopDate, in which case, the +1 was needed for that?

Reply:
Yes, correct
It was failing because of this input validation check startDate should be < stopDate thats why for Today date range startDate always set with undefined due to which it got fail so added  +1 .

Reply:
The data value is set as 'max="Tue Sep 19 2023 12:29:14 GMT+0530 (India Standard Time)1"' instead of '2023-09-19.' It just concatenated the '1' and didn't add the day.

The AS-IS behavior is that if '2023-09-19' is set for the start and end date, then the 'min' and 'max' are also set to '2023-09-19.' So, I don't think we need to add one day.

@Samridhi-98 @gsluthra @priyanshu1602